Source/Private/_ASCoreAnimationExtras.mm

```diff
@@ -98,29 +98,29 @@ extern void ASDisplayNodeSetResizableContents(id<ASResizableContents> obj, UIIma
 
 NSString *ASDisplayNodeNSStringFromUIContentMode(UIViewContentMode contentMode)
 {
-  for (int i=0; i< ARRAY_COUNT(UIContentModeDescriptionLUT); i++) {
-    if (UIContentModeDescriptionLUT[i].contentMode == contentMode) {
-      return UIContentModeDescriptionLUT[i].string;
+  for (auto &e : UIContentModeDescriptionLUT) {
```

Comment:
Maybe a better variable name as `e` in this cases ;)

Reply:
I think `e` is short for `entry` and maybe there's a better name out there but I don't really want to wait for CI to rebuild.